src/scriptwriter/scenarios/ScenariosHandler.js

```diff
@@ -110,4 +114,21 @@ export default class ScenariosHandler {
 
         return scenario;
     }
+
+    /**
+     * Loads a start url for scenario from specified url paths
+     * @returns {string} start url
+     */
+    _getRandomScenarioStartUrl() {
+        let scenarios = this._config.urlPaths;
+
+        if (this._index >= scenarios.length) {
+            this._index = 0;
+        }
+
+        let path = scenarios[this._index];
```

Comment:
These 3 lines could be simplified into a still readable one liner.

```javascript
return this._config.url + urlPaths[this._urlPathsIndex++];
```

Reply:
Done